trilogy/trilogy.py

```
@@ -1,1322 +1,792 @@
+
+    if invert:
+        data = 255 - data
+
+    n_channels = data.shape[-1]
+    if n_channels == 3:
+        img = Image.fromarray(data, mode="RGB")
+    elif n_channels == 1:
+        img = Image.fromarray(data[:, :, 0], mode="L")
     else:
-        iext = 0
+        raise ValueError(f"Unexpected number of channels: {n_channels}")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)